android/src/main/java/com/oblador/keychain/KeychainModule.java

```diff
@@ -164,6 +239,28 @@ public void resetInternetCredentialsForServer(@NonNull String server, ReadableMa
         resetGenericPasswordForOptions(server, promise);
     }
 
+    @ReactMethod
+    public void canCheckAuthentication(ReadableMap options, Promise promise) {
```

Comment:
Now that this method is implemented for Android, the extra check in `index.js` can safely be removed.

Reply:
done